src/services.js

```diff
@@ -0,0 +1,18 @@
+async function postData(url = '', data = {}) {
+    // Default options are marked with *
+    const response = await fetch(url, {
```

Comment:
- [ ] Great work so far, your app works as expected but it would be better to arrow functions instead of the function keyword since it is recent and it solves some unforeseen issues. So for your async function here you could have something like this:
- Arrow syntax:
![Annotation 2022-03-02 145821](https://user-images.githubusercontent.com/85113567/156375688-18cb61d3-b658-4597-b05d-d05e28a4af57.jpg)
- Kindly change all your functions to arrow functions 💯